client/src/components/AssignPositionModal.js

```diff
@@ -101,12 +104,75 @@ const AssignPositionModal = ({ person, showModal, onCancel, onSuccess }) => {
           This position is currently held by{" "}
           <LinkTo modelType="Person" model={position.person} />. By selecting
           this position, they will be removed.
+          {person.position.type !== position.type ? (
+            <>
+              &nbsp;Permission of <b>{position.name}</b> position is going to be
```

Comment:
If you want to force a space, use `{" "}` instead of `&nbsp;` (i.e. a regular space, to allow the browser to break up the sentence if needed).